ParisBNF/abb/BNFabb24.xml

```diff
@@ -165,7 +165,7 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                             <list>
                                   <item xml:id="a1">
                                     <locus from="4ra" to="4rb"/>
-                                      <desc type="GuestText" corresp="LIT5549TribesApostles">The Discourse of Dionysius on the Tribes of the Apostles.</desc>   
+                                      <desc type="GuestText"><title type="complete" ref="LIT5549TribesApostles"/>The Discourse of Dionysius on the Tribes of the Apostles.</desc>   
```

Comment:
`<title type="complete" ref="LIT5549TribesApostles">The Discourse of Dionysius on the Tribes of the Apostles</title>.`

Reply:
ok, hope I corrected it rightly.